src/auth/hooks/useAuthProvider.ts

```diff
@@ -56,16 +57,24 @@ export function useAuthProvider({
   }, [authenticating]);
 
   useEffect(() => {
-    if (!authenticated && !authenticating) {
+    if (!authenticated && !authenticating && permitCredentialsAPI) {
+      setPermitCredentialsAPI(false);
       loginWithCredentialsManagementAPI(handleLogin);
     }
   }, [authenticated, authenticating]);
 
   const userDetails = useQuery<UserDetails>(userDetailsQuery, {
     client: apolloClient,
-    skip: !authenticated
+    skip: !authenticated,
+    fetchPolicy: "cache-and-network"
   });
 
+  useEffect(() => {
+    if (authenticated) {
+      setPermitCredentialsAPI(true);
```

Comment:
Would it make more sense to trigger this on user logout? (inside `isCredentialsManagementAPISupported` if statement)

Reply:
Not sure, because what if token expires? It won't be logout this way.